frame/election-provider-multi-phase/src/unsigned.rs

```diff
@@ -1381,7 +1423,11 @@ mod tests {
 			let compact_clone = compact.clone();
 
 			// when
-			MultiPhase::trim_assignments_length(encoded_len as u32 - 1, &mut assignments, encoded_size_of).unwrap();
+			MultiPhase::trim_assignments_length(
+				encoded_len as u32 - 1,
```

Comment:
is encoded len guaranteed to be greater than zero?

Reply:
In this test, yes. However, given that this is within test code, it should be ok to fail the test if that assumption is ever invalidated.